apps/catalog/catalog/src/app/marketplace/offer/shell/contract-view/contract-view.component.scss

```
@@ -3,10 +3,14 @@
   margin: 36px 48px;
 }
 
-a {
+a [mat-icon-button] {
```

You need to glue them together: https://www.w3schools.com/cssref/sel_attribute.asp
```suggestion
a[mat-icon-button] {
```